datasets/bg/jud_declarations/crawler.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, Generator, List, Optional
+from lxml.html import HtmlElement
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise, OccupancyStatus
+
+
+def parse_html_table(
+    table: HtmlElement,
+    skiprows: int = 0,
+    headers: Optional[List[str]] = None,
+) -> Generator[Dict[str, HtmlElement], None, None]:
+    for rownum, row in enumerate(table.findall(".//tr")):
+        if rownum < skiprows:
+            continue
+        cells = row.findall("./td")
+        yield {hdr: c for hdr, c in zip(headers, cells)}
+
+
+def crawl_row(context: Context, row: Dict[str, HtmlElement], doc: HtmlElement):
+    doc.make_links_absolute(context.data_url)
```

Comment:
I'd do this in the loop, then it only happens once per page, and you don't have to pass `doc` to `crawl_row`